homeassistant/components/recorder/pool.py

```diff
@@ -131,7 +131,7 @@ def _do_get(self) -> Any:
         if DEBUG_MUTEX_POOL:
             _LOGGER.debug("%s wait conn%s", threading.current_thread().name, trace_msg)
         # pylint: disable-next=consider-using-with
-        got_lock = MutexPool.pool_lock.acquire(timeout=1)
+        got_lock = MutexPool.pool_lock.acquire(timeout=10)
```

Comment:
Unsure if this could have triggered, But spotted during investigation. We can't assume 1 second will be enough for a competing task to complete.

Reply:
10 is probably long than needed but likely not hurting anything